### ISSUE TITLE
Run Adminator with Electron.js

### ISSUE THREAD

**Issue by YoussefRekik** [puikinsh/Adminator-admin-dashboard#43]
Hello I would like to thank you for this amazing template ! 
Is there a way to run it with Electron.js ?
I saw mentions of SQLite when installing it but I didn't find any SQLite related code nor examples

**Comment by AndreiCN** [comment]
Unfortunately we didn't run it with Electron. For better results we strongly suggest to use the template as it is by default. If you have no further questions please close the ticket, thank you !

**Comment by jepperaskdk** [comment]
From the dependency-tree, it looks like webpack-dashboard installs some sqlite-functionality:
```
├─┬ webpack-dashboard@1.1.1
│ ├-┬ inspectpack@2.2.2
│ │ ├─┬ better-sqlite3@4.1.0
```

**Comment by krzysiekpiasecki** [comment]
> Is there a way to run it with Electron.js ?

@YoussefRekik 

Yes. It's simple, but unfortunately the build is still broken (will be fixed with merge #48. So, from the root of the project run the shell command: ```electron ./build/index.html```

You will find more details how to run electron app here: https://electronjs.org/docs/tutorial/quick-start. The main files are: ```package.json, index.html, index.js```. The documentation mentions about main.js but latest electron CLI uses index.js 

```Electron 1.8.2 - Build cross platform desktop apps with JavaScript, HTML, and CSS
  Usage: electron [options] [path]

  A path to an Electron app may be specified. It must be one of the following:
    - index.js file.
    - Folder containing a package.json file.
    - Folder containing an index.js file.
    - .html/.htm file.
    - http://, https://, or file:// URL.

  Options:
    -d, --default         Run the default bundled Electron app.
    -i, --interactive     Open a REPL to the main process.
    -r, --require         Module to preload (option can be repeated).
    -v, --version         Print the version.
    -a, --abi             Print the Node ABI version.
```

May I ask you to close this ticket, if you haven't got any further questions.
